hare/integration_test.go

```diff
@@ -55,7 +55,7 @@ func Test_16Nodes_HareIntegrationSuite(t *testing.T) {
 		output := make(chan TerminationOutput, 1)
 		signing := NewMockSigning()
 		oracle.Register(true, signing.Verifier().String())
-		proc := NewConsensusProcess(cfg, *instanceId1, his.initialSets[idx], oracle, signing, s, output)
+		proc := NewConsensusProcess(cfg, *instanceId1, his.initialSets[idx], oracle, signing, s, output, log.NewDefault("ConsensusProcess"))
```

Comment:
I'd use a logger name which is more idicative to the instance and test. this will help a lot more than reading logs from "ConsensusProcess"

Reply:
Fixed